src/openrct2/windows/ride_list.c

```diff
@@ -260,6 +266,10 @@ static void window_ride_list_mouseup(rct_window *w, int widgetIndex)
 	case WIDX_OPEN_LIGHT:
 		window_ride_list_open_all(w);
 		break;
+	case WIDX_QUICK_DEMOLISH:
+		_quick_demolish_mode ^= 1;
```

Comment:
>_quick_demolish_mode = !_quick_demolish_mode

I've changed it, thank you. Is there any specific differences between "_quick_demolish_mode ^= 1;" and "_quick_demolish_mode = !_quick_demolish_mode;" aside from legibility differences? 
it seems like a "^= 1" would say to flip whatever the value is reguardless of it, where as the _quick_demolish_mode = !_quick_demolish_mode" would say to read the value and then assign to that same value the opposite. But I'm also not as knowledged on this field as you are.